polyply/data/oplsaaLigParGen/citations.bib

```diff
@@ -35,3 +35,11 @@ @article{OPLSLigParGen3
   year={2017},
   doi={10.1021/acs.jpcb.7b00272}
 }
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```

Comment:
following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```